recipes/underworld/meta.yaml

```
@@ -0,0 +1,84 @@
+{% set name = "underworld" %}
+{% set version = "2.14.1" %}
+{% set sha256 = "ec922eb6d10d6a6031d5c76b106df0c79b122ee61c43900866f2017bf819b2eb" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/ad/44/e491e7faa66cd927b722304d3e0a7ecc5f6c24cefa6044868a84548303d5/underworld-{{ version }}b0.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: true  # [win or osx]
+  number: 0
+  script: PETSC_DIR=$PREFIX {{ PYTHON }} -m pip -vv install --no-deps .  
+  # shared objects to ignore when checking linkage information
+  missing_dso_whitelist:
+    - "**/site-packages/underworld/lib/**"
+
+
+requirements:
+
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ mpi }}
+    - petsc {{ petsc }}
+    - pkg-config
+    - cmake
+    - swig >=4.0.0
+
+  host:
+    - pip
+    - setuptools
+    - python
+    - libxml2
+    - numpy >=1.18
+    - ninja
+    - petsc {{ petsc }}
+    - {{ mpi }}
+
+  run:
+    - python
+    - {{ mpi }}
+    - numpy
+    - petsc {{ petsc }}
+    - pint
+    - scipy
+    - mpi4py
+    - h5py
+
+test:
+  imports:
+    - underworld
+
+  requires:
+    - pytest
+    - matplotlib
```

```suggestion
    - matplotlib-base
```

you probably don't want the full matplotlib (which pulls in qt).

Do you need matplotlib for your simple test? If so, then it should be a requirement.

Otherwise, maybe remove it from the test section.

---

Don't need matplot lib in the tests. But we need it for the run section. Adding it there.